### PR TITLE
[Enhancement] Start timer only after initial button press and display seconds left and average result 

### PR DESCRIPTION
**Pull request by Tauka**
* I think this will make game experience richer, cause now you would have time to mentally prepare yourself before start (honestly, I expected this behavior by default).
* Also I would like to know my average score, so If I fail before timer expires, I would know what score I would ideally reach

### CODE REVIEW

**Comment by ruskakimov** [comment]
also you can deploy your changes by

1. installing surge first (`npm i -g surge`)
2. changing the domain name in deploy script (`package.json`)
3. running `npm run deploy` or `yarn run deploy`

**Comment by Tauka** [comment]
#### I made requested changes, please review them
> now `Expected score` displays `average * 60`, which is the score player would get if completed the game in the same pace

**Comment by ruskakimov** [comment]
Could you also add current score to the overlay? :) It feels like it's missing. After I will merge and deploy it.

**Comment by Tauka** [comment]
* Added `Your Score` to overlay
* Added `Go to Main` button
> This was done in order to avoid accidentally clicking away overlay during intensive gameplay

File: src/components/GameScreen/index.js [hunk 1]
```diff
@@ -54,10 +54,23 @@ const OverlayChild = styled.div`
...
+const OverlayButton = styled.button`
+  border: none;
+  background-color: green;
+  border-radius: 5px;
+  padding: 10px 20px;
+  font-size: 1.2rem;
+  font-weight: bold;
+  cursor: pointer;
+  /* color of button will be the same as the color of the last word */
+  background-color: ${({ color }) => color ? color : "#3764a9"};
```
Comment (ruskakimov) [inline src/components/GameScreen/index.js]: is this condition necessary? `color` should always be set to something

Comment (Tauka) [reply src/components/GameScreen/index.js]: Ideally yes, but it's a safety measure. As I am not aware of all effects of this application, I assumed that sometimes, for some reasons, color might get `undefined`. If you are sure it won't happen under any circumstances, I can remove condition

Comment (ruskakimov) [reply src/components/GameScreen/index.js]: yep, remove it please. :) Other components use that value directly.

Comment (ruskakimov) [reply src/components/GameScreen/index.js]: after that i can merge it

**Comment by Tauka** [comment]
@ruskakimov Btw, how about adding keyboard support? So you can play it with left and right arrows, my phone does not support anything 😄

**Comment by ruskakimov** [comment]
@Tauka you can do it if you want :) But better do it in a separate PR. I can even create an issue.

**Comment by Tauka** [comment]
#### Changes has been made

**Pull request merged**